src/module.c

```diff
@@ -122,7 +140,8 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
 	AR_RegisterFuncs();      // Register arithmetic functions.
 	Agg_RegisterFuncs();     // Register aggregation functions.
 	_PrepareModuleGlobals(); // Set up global lock and variables scoped to the entire module.
-	RegisterForkHooks();     // Set up forking logic to prevent bgsave deadlocks.
+	RegisterForkHooks(
+		ctx);  // Set up hooks for key rename and forking logic to prevent bgsave deadlocks.
```

Comment:
Make sure this all fits in a single line

Reply:
done